activemodel/test/cases/normalized_attribute_test.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "models/aircraft"
+require "active_support/core_ext/string/inflections"
+
+class NormalizedAttributeTest < ActiveModel::TestCase
```

Comment:
Since we're testing the core behavior here now, would it be worth removing duplicated tests from `activerecord/test/cases/normalized_attribute_test.rb`?  (Though, obviously, database-related tests should be kept there.)